lsp/server/src/server.ts

```diff
@@ -30,10 +34,21 @@ let hasConfigurationCapability = false;
 let hasWorkspaceFolderCapability = false;
 export let hasDiagnosticRelatedInformationCapability = false;
 
+let extensionId: string = '';
```

Comment:
Can we get rid of explicit typing like `: string` when we are initializing it to empty string? In TypeScript, implicit type declaration (also known as type inference) occurs when you declare a variable and assign it a value without explicitly specifying the type. TypeScript automatically infers the type based on the assigned value.

Reply:
done